src/worker/core/player/genRatings.basketball.ts

```diff
@@ -183,6 +183,35 @@ const genRatings = (
 		skills: [],
 	};
 
+	// Youngest prospects !== 19 will be scaled, scaling stops at age 14 and 30
+	let age = g.get("draftAges")[0];
+	age = helpers.bound(age, 14, 30);
+	if (age !== 19) {
+		const scale = 3 * (19 - age);
+		const rtgs = [
+			"stre",
+			"endu",
+			"ins",
+			"dnk",
+			"ft",
+			"fg",
+			"tp",
+			"oiq",
+			"diq",
+		] as const;
+		// These ratings develop slowly compared to others, so they scale less. Works well in testing
+		const rtgsDevelopSlow = ["spd", "jmp", "drb", "pss", "reb"];
```

Comment:
Don't fix it, I'm doing it, but... these are never getting applied!

Reply:
bruh